BridgeAppSDK/Localization.swift

```diff
@@ -49,15 +53,81 @@ class Localization: NSObject {
         return str
     }
     
-    // List of the strings used in this module. This func is never called but is included to allow running 
-    // the localization macro
-    static var listOfAllLocalizedStrings: [String : String] = {
+    
+    // MARK: Localized App Name
+    
+    public static var localizedAppName : String = {
+        let mainBundle = NSBundle.mainBundle()
+        let bundleInfo = mainBundle.infoDictionary
+        let localizedBundleInfo = mainBundle.localizedInfoDictionary
+        
+        func trim(obj: AnyObject?) -> String? {
+            guard let str = obj as? String else { return nil }
+            let result = str.stringByTrimmingCharactersInSet(NSCharacterSet.whitespaceAndNewlineCharacterSet())
+            guard result != "" else { return nil }
+            return result
+        }
+        
+        let result =
+            trim(bundleInfo? ["CFBundleDisplayName"]) ??
+            trim(bundleInfo? ["CFBundleName"]) ??
+            trim(bundleInfo? ["CFBundleExecutable"]) ??
+            trim(mainBundle.objectForInfoDictionaryKey("CFBundleDisplayName")) ??
+            trim(mainBundle.objectForInfoDictionaryKey("CFBundleName")) ??
+            trim(mainBundle.objectForInfoDictionaryKey("CFBundleExecutable")) ??
+            trim(localizedBundleInfo? ["CFBundleDisplayName"]) ??
```

Comment:
Good point. This was a copy/paste from AppCore that I included because it seemed to be that not all apps were defining the app name in the expected place. I tilled my head and then got distracted and forgot about it. I'll rearrange.